docs/host_install.md

```diff
@@ -231,7 +215,7 @@ We recommend running a ledger instance locally in the provided Docker image:
 ```
 cd $PDO_SOURCE_ROOT
 mkdir -p $PDO_LEDGER_KEY_ROOT
-make -C docker test-env-setup(-ccf)
+make -C docker test-env-setup(-ccf-only)
```

Comment:
is this a runnable command (with the parenthesis)? -- I guess even before it was not right.

Reply:
thank you. I will put it as a separate command. parenthesis not to be used as part of command.